frontend/components/Players/DraggableQueue.vue

```diff
@@ -85,6 +89,23 @@ export default Vue.extend({
       }
     }
   },
+  /**
+   * Scroll the currently playing item to view after opening the queue
+   */
+  mounted() {
+    const ref = this.$refs.listItems as Vue[];
+    const currentItemId = this.playbackManager.getCurrentItem?.Id || '';
+
+    const el = ref.find(
+      (v) => v.$vnode.key === `${currentItemId}-${ref.indexOf(v)}`
+    );
+
+    if (el?.$el) {
+      window.requestIdleCallback(() => {
```

Comment:
Why do we use it? I'm only curious I trust you there's a reason ^^

Reply:
Good catch!

I wanted to document it, but forgot about it while testing. I added the comment directly above the function, but here's for reference as well:

```
/**
       * As the queue opening has a transition effect, el.$el.scrollIntoView() doesn't work directly,
       * as the parent DOM node is not fully rendered while the transition is taking place
       * (so scrollIntoView() doesn't know exactly what to scroll).
       *
       * The browser always give full priority to the DOM manipulation and painting process,
       * while the idle callback runs with lower priority. This assures us that the view will be scrolled to
       * the currently playing element as soon as all the DOM operations and transitions are over.
       */
```